app/_src/gateway/kong-enterprise/secrets-management/advanced-usage.md

```diff
@@ -128,8 +128,8 @@ Most of the vaults also support secret rotation by using TTLs:
 
 Parameter | Field name | Description
 ----------|------------|------------
-`vaults.config.ttl` | **TTL** | Time-to-live (in seconds) of a secret from the vault when cached by this node.
-`vaults.config.neg_ttl` | **Negative TTL** | Time-to-live (in seconds) of a vault miss (no secret).
-`vaults.config.resurrect_ttl` | **Resurrect TTL** | Time (in seconds) for which stale secrets should be resurrected forwhen they cannot be refreshed (e.g., the vault is unreachable).
+`vaults.config.ttl` | **TTL** | Time-to-live (in seconds) of a secret from the vault when cached. The special value of 0 means "no rotation", and it is the default. When using non-zero values, it is recommended that they are at least 1 minute.
```

Comment:
```suggestion
`vaults.config.ttl` | **TTL** | Time-to-live (in seconds) of a secret from the vault when it's cached. The special value of 0 means "no rotation" and it's the default. When using non-zero values, it is recommended that they're at least 1 minute.
```
For the "it is recommended that they're at least 1 minute", does that mean they should be in minute increments? So "2" is okay but not "2.6"? Or just that they shouldn't be anything less than "1"?